### DISCUSSION TITLE
vscode extension

### DISCUSSION THREAD

**Discussion by tigawanna** [Tencent/cherry-markdown#718] [Q&A]
Hey guys, is this team responsible for the vscode extension because it would be perfect if it let us change language to English l, I can't find the save button, I would really enjoy that editing experience but in English

**Comment by RSS1102** [comment]
We are considering enjoying the holiday and will consider resolving this issue after the holiday ends. Thank you for your feedback.😀😀

**Comment by tigawanna** [comment]
hope you had great holidays , just checking in on this one
